sql/src/main/java/io/crate/protocols/postgres/PGType.java

```diff
@@ -54,7 +67,7 @@ private PGType(int oid,  int typeLen, int typeMod, int formatCode) {
         final static int OID = 1043;
 
         StringType() {
-            super(OID, -1, -1, 0);
+            super(OID, -1, -1, FormatCode.BINARY);
```

Comment:
the first one is the typeLen (-1 is variable length, as in `pg_type.typlen`)
the second one is a type modifier (as in `pg_attribute.atttypmod`)